tiny_dnn/core/kernels/cl_kernels/activation.cl

```diff
@@ -0,0 +1,108 @@
+#ifndef __OPENCL_VERSION__
```

Comment:
I think it's better to move these files into third_party/caffe for better maintenance.